scripts/nb-tester/requirements.txt

```
@@ -1,7 +1,7 @@
 # We pin to exact versions for a more reproducible and
 # stable build.
 
-qiskit[all]==2.0.0rc2
+qiskit[all]==2.0.0
```

```suggestion
qiskit[all]~=2.0.0
```

Just following the convention of pinning to the minor version rather than the patch